File: backend/src/main/java/com/coing/domain/bookmark/controller/dto/BookmarkUpdateRequest.java
package com.coing.domain.bookmark.controller.dto;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;

public record BookmarkUpdateRequest(
	@NotNull(message = "{bookmarkId.required}") Long bookmarkId,
	@NotBlank(message = "{code.required}") String code
) {
}

File: backend/src/main/java/com/coing/domain/bookmark/controller/dto/BookmarkUpdateRequest.kt
package com.coing.domain.bookmark.controller.dto

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.NotNull

data class BookmarkUpdateRequest(
    @field:NotNull(message = "{bookmarkId.required}")
    val bookmarkId: Long,

    @field:NotBlank(message = "{code.required}")
    val code: String
)
